tox.ini

```diff
@@ -25,10 +24,12 @@ deps=
 [testenv:lint]
 basepython=python3.6
 deps=
+    black
     flake8>=2.2.0
     isort>=4.2.15
 commands=
-    flake8 .
+    black --check flags setup.py
+    flake8 flags setup.py
```

Comment:
Why specify `flags setup.py`? Just to avoid looking over a bunch of non-Python files?

Reply:
Basically.